.github/actions/setup-ccache/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Setup ccache
+inputs:
+  extra-cache-key:
+    required: false
+    default: ''
+runs:
+  using: composite
+  steps:
+    - name: Get Date
+      id: get-date
+      run: echo "::set-output name=today::$(/bin/date -u '+%Y%m%d')"
+      shell: bash
+
+    - name: Setup ccache
+      uses: hendrikmuhs/ccache-action@2181be813387616fc2b8dca72d3ff8b912b25f73
```

Comment:
Can you reference a version instead of a git hash?

Reply:
Did this for security purposes but I'm happy to change it if you're not concerned about that.

Reply:
Shouldn't a v1.x be stable?

Reply:
I was worried more about security than stability. Tags can be force pushed with malicious code.

Reply:
Should we not trust this particular action author?

I understand the security concern, but since we also reference tags in other actions, to potentially pick up bug fixes/performance improvements), IMO that's a worthy tradeoff. Open to other core team opinion